netbeans.apache.org/src/content/templates/footer.gsp

```diff
@@ -88,8 +88,9 @@
                 <ul>
                     <li><a href="/download/index.html#releases">Releases</a></li>
                     <ul>
-                        <li><a href="/download/nb90/nb90-beta.html">Apache NetBeans 9.0 (beta)</a></li>
+                        <li><a href="/download/nb90/nb90.html">Apache NetBeans 9.0</a></li>
```

Comment:
I wonder if it's important to link to the RC and beta once we have a proper release?

Reply:
Not important, but gives an idea of the work done :-)

Reply:
agree with Antonio here